City-of-Helsinki/drupal-hdbt
# UHF-11059: Home care client fee calculator additions

# [UHF-11059](https://helsinkisolutionoffice.atlassian.net/browse/UHF-11059)


## What was done


* Added new text that appears when the safetyphone selection is selected.

## How to install

* Make sure your SOTE instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-11059`
* Run `make drush-cr`

## How to test


* [ ] Go to https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/senioripalvelut/kotihoito/kotihoidon-asiakkaaksi/kotihoidon-asiakasmaksun-laskuri and make sure if you select the "Safety phone and bracelet" to be added to the calculations a text is shown under the selection that says:

> Please note that an emergency visit from the safety services costs EUR 23.25 or EUR 46.50 per visit, depending on your income limits. Each month, you will be billed for a maximum of five emergency visits, totalling no more than EUR 116.25 or EUR 232.50, depending on your income limits.

* [ ] Make sure it works for all three languages.

## Continuous documentation



## Translations
